lib/pf/Switch/Cisco/Catalyst_2950.pm

```diff
@@ -837,25 +837,33 @@ sub setTaggedVlans {
     my $OID_vlanTrunkPortVlansEnabled3k = '1.3.6.1.4.1.9.9.46.1.6.1.1.18';
     my $OID_vlanTrunkPortVlansEnabled4k = '1.3.6.1.4.1.9.9.46.1.6.1.1.19';
 
-    my @bits = split //, ("0" x 1024);
+    my @bits = split //, ("0" x 4096);
     foreach my $t (@vlans) {
-        if ($t > 1024) {
-            $logger->warn("We do not support Tagged Vlans > 1024 for now on Cisco switches. Sorry... but we could! " .
+        if ($t > 4096) {
```

Comment:
I think we can remove that statement... otherwise it's switch vendors that will contact us to enhance that capability in their product ;-)

Reply:
Agree with @extrafu, that statement can never be true unless we f*cked up or Cisco starts supporting VLANs above 4096 which will likely require much more work than adjusting this sub

Reply:
https://github.com/inverse-inc/packetfence/commit/0b9e82b33139eb2669817883c86cb7261dc3e222